docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -39,13 +56,21 @@ Cause Model Diagram
 
 Model Assumptions and Limitations
 ---------------------------------
+This model is designed to be used for estimating DALYs due to LRI that are averted from a
+country-level intervention(e.g. food fortification or supplementation given to a percentage of the population)
+that can reduce IRI incidence as a downstream effect.
+
 There is substantial additional effort in GBD to divide LRI
-burden into the aetiologies of LRI, but we have not included
+burden into the aetiologies of LRI, but for the use of this model, we do not included
 aetiologies in this simple model.
 
 There are three sequelae associated with LRI. We do not consider Guillain-Barré syndrome
```

Comment:
@XiaoluQianUW I just noticed that here and in the State Data table, it says we are not including sequela s671 (Guillain-Barre due to LRI). I believe that we *do* want to include this sequela (and that is what the Vivarium model currently does, correct, @collijk ?).

The model limitation is that we are not tracking the long-term effects of this sequela (which can include paralysis, for example), but are only counting its YLDs *while the simulant is experiencing a bout of LRI*. The point is that since the prevalence of GBS is so low, there would probably not be much benefit in attempting to capture the long-term YLDs in addition to the short term YLDs.

@beatrixkh Does this match your understanding for diarrheal diseases (which also has GBS as a sequela)?

Reply:
For diarrhea, both GBD and we are including GB as one of the sequelae (the other three being mild, moderate, and severe diarrhea). This is because this etiology involves one of the pathogens that can cause diarrhea also causing GB, so if one could avert that diarrhea case, one would also have averted the GB case, if I understand correctly.

GB can be life threatening--but can also be nonfatal, and yes, contribute to significant long-term nonfatal burden, so yes, stopping the model after 5 years does fail to capture those long-term YLDs averted.

I believe the mechanism for LRI--> GB is similar, so yes, it would make sense to include it in this model!

Reply:
Thanks for the clarification, @beatrixkh ! I'm still a bit confused about whether the three sequelae are mutually exclusive, though in your other comment you indicate this has something to do with como...

Reply:
I have talked with Beatrix and I believe the three sequelae are not mutually exclusive. I was able to pull the prevalence for GBS from GBD output. It is a very small number though but still exists.